pygorithm/Backtracking/N-Queen.py

```diff
@@ -0,0 +1,89 @@
+# Python program to solve N Queen 
+# Problem using backtracking 
+
+global N 
+N = 4
+
+def printSolution(board): 
+	for i in range(N): 
+		for j in range(N): 
+			print (board[i][j]), 
+		print
+
+
+# A utility function to check if a queen can 
+# be placed on board[row][col]. Note that this 
+# function is called when "col" queens are 
+# already placed in columns from 0 to col -1. 
+# So we need to check only left side for 
+# attacking queens 
+def isSafe(board, row, col): 

```

Comment:
Change function name to `is_safe` so as follow Python PEP8 styling

Reply:
Done